presto-orc/src/main/java/com/facebook/presto/orc/reader/MapFlatSelectiveStreamReader.java

```diff
@@ -263,7 +265,9 @@ private void readNotAllNulls(int offset, int[] positions, int positionCount)
         int streamPosition = 0;
 
         int[] nonNullPositions = new int[positionCount];
+        int[] nullPositions = new int[positionCount];
```

Comment:
nit: we should be reusing this array; I do see that we are not reusing nonNullPositions array though; let's fix both in some future PR